### DISCUSSION TITLE
Resume Bundle X ATS Scoring

### DISCUSSION THREAD

**Discussion by kartikmehta18** [recodehive/Stackoverflow-Analysis#328] [Q&A] [answered]
[Kartik's Resume1.pdf](https://github.com/user-attachments/files/16227573/Kartik.s.Resume1.pdf)

**Comment by HemantSachdeva** [comment]
[Hemant_Sachdeva.pdf](https://github.com/user-attachments/files/16322983/Hemant_Sachdeva.pdf)
I acknowledge that one pager resume is ideal for a lot of people reviewing it but I just can't figure it out what's not worth mentioning :/

**Reply by sanjay-kv** [reply]
[reviewHemant_Sachdeva.pdf](https://github.com/user-attachments/files/16323023/reviewHemant_Sachdeva.pdf)
The above is some improvements you need to do


[ATS Version of CV (7).docx](https://github.com/user-attachments/files/16323026/ATS.Version.of.CV.7.docx)
the above is ATS version of it, its how ATS read your resume, so if any alignment graphics can affect the content

**Comment by FarhaKousar1601** [comment]
[Farha Kousar.pdf](https://github.com/user-attachments/files/16323801/Farha.Kousar.pdf)

[https://github.com/FarhaKousar1601](url)

@sanjay-kv

**Comment by HeySkidee** [comment]
> [Kartik's Resume1.pdf](https://github.com/user-attachments/files/16227573/Kartik.s.Resume1.pdf)

i heard a lot of people say that resumes with this kind of layout are not ats friendly + never add your photo in your resume

**Reply by hprasadsakhare** [reply]
![resume ats](https://github.com/user-attachments/assets/f3edc526-7ec2-48b0-b65d-19b4f7bdba77)

You should prefer this type of resume template for a high ATS score and an optimized resume.

**Comment by HeySkidee** [comment]
[Harshit Kumar Ojha Resume.pdf](https://github.com/user-attachments/files/16458310/Harshit.Kumar.Ojha.Resume.pdf)

Here is my resume. Any feedback is appreciated

**Comment by yazansedih** [comment]
The resume seems good.

**Comment by matiborda** [comment]
Resume seems good

**Comment by DarkHacker28** [comment]
Use :

https://resumeworded.com/resume-scanner


and use good templete

**Comment by Richajaishwal0** [comment, marked as answer]
Guys, you can easily improve your ATS score using Resume Worded. Just upload your resume, and it will evaluate it out of 100 while giving you detailed feedback. Although the free trial has some limitations, the insights are very useful for boosting your score.

Here’s what I did:
Uploaded my resume to Resume Worded and checked the feedback.
Took that feedback and gave it to ChatGPT with a prompt — not to change the content, but to optimize keywords, refine bullet points, remove unnecessary buzzwords etc. based on feedback. 
Updated my resume based on the suggestions and re-uploaded it.
Each time, I noticed my ATS score improved.
